devel/ci/bodhi-ci

```diff
@@ -299,6 +299,15 @@ def integration_build(concurrency, container_runtime, failfast, pyver, release,
     _run_jobs(_build_jobs_list('integration-build', concurrency, release, pyvers=pyver))
 
 
+@cli.command("integration-download")
+@concurrency_option
+@failfast_option
+@tty_option
+def integration_download(concurrency, failfast, tty):
+    """Download the database dumps for integration testing."""
+    _run_jobs(_build_jobs_list('integration-download', concurrency, releases=[], pyvers=[3]))
```

Comment:
You could also have defined the two jobs right here, as with the `clean` command. This way you wouldn't have to add conditionals to the `_build_jobs_list` function.

Reply:
On the other hand, it means instantiating the classes twice, which is kind of against DRY. Not sure which is the lesser evil.

Reply:
Yeah I kinda like having once source of setting up those classes.